be/src/runtime/tablets_channel.cpp

```diff
@@ -242,7 +242,7 @@ Status TabletsChannel::_open_all_writers(const PTabletWriterOpenRequest& request
         wrequest.tuple_desc = _tuple_desc;
         wrequest.slots = index_slots;
         wrequest.is_high_priority = _is_high_priority;
-        wrequest.ptable_schema_param = request.schema();
+        wrequest.table_schema_param = _schema;
```

Comment:
`_schema` is inited from `request.schema()`, and it's life cycle is longer than delta writers, so we can use `_schema` directly to avoid copying the schema twice(the first time is here, the second time is in the ctor of DeltaWriter)